File: python/Check.py
def check(list1, val):
    for x in list1:
        if val >= x:
            return "Yes"
    return "No"


list1 = [10, 20, 30, 40, 50, 60]
val = 5
print(check(list1, val))
val = 20
print(check(list1, val))

File: java/Check.java
import java.util.Scanner;
class Check
{
 public static void main(String args[])
 {Scanner sc=new Scanner(System.in);
  System.out.print("enter the no tobe checked");
  int n=sc.nextInt();
  if(n%2==0)
    System.out.println("EVEN");
  else
    System.out.println("ODD");
  sc.close();    

 }   
}